src/pages/DistributionsPage/DistributionsPage.test.tsx

```diff
@@ -72,26 +70,14 @@ jest.mock('./queries', () => ({
   useSubmitDistribution: jest.fn(),
 }));
 
-let queryClient: QueryClient;
```

Comment:
removed this code because it didn't fix the sporadic Jest crash that happens when running this test file